config/software/rabbitmq.rb

```diff
@@ -23,27 +23,78 @@
 
 dependency "erlang"
 
-version("3.7.6")  { source md5: "d8db2d979a66e520b23be29d9b43af70" }
-version("3.6.15") { source md5: "1dd9a5d000e11064efec23c1a76e5d5c" }
-version("3.6.0")  { source md5: "61a3822f3af0aaa30da7230dccb17067" }
-version("3.3.4")  { source md5: "61a3822f3af0aaa30da7230dccb17067" }
-version("2.8.7")  { source md5: "35e8d78f8d7ae4372db23fe50db82c64" }
 version("2.7.1")  { source md5: "34a5f9fb6f22e6681092443fcc80324f" }
```

Comment:
We can probably nuke 2 at this point

Reply:
Will do.

Reply:
Actually, rabbitmq should be going away soon, so hopefully we nuke the entire thing at some point.